### PR TITLE
Use the new-ish frontend_title field of the profile for the menu title

### PR DESCRIPTION
**Pull request by adixon**
As per https://lab.civicrm.org/dev/drupal/-/issues/132

### CODE REVIEW

File: src/Plugin/Derivative/LocalTasks.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function getDerivativeDefinitions($base_plugin_definition) {
     foreach ($uf_groups as $key => $uf_group) {
       if ($uf_group['is_active']) {
         $this->derivatives["civicrm.{$key}"] = $base_plugin_definition;
-        $this->derivatives["civicrm.{$key}"]['title'] = $uf_group['title'];
+        $this->derivatives["civicrm.{$key}"]['title'] = $uf_group['frontend_title'];
```
Comment (seamuslee001) [inline src/Plugin/Derivative/LocalTasks.php]: @adixon this broadly is heading down the right path but I would suggest doing this instead

```suggestion
        $this->derivatives["civicrm.{$key}"]['title'] = !empty($uf_group['frontend_title']) ? $uf_group['frontend_title'] : $uf_group['title'];
```

Comment (adixon) [reply src/Plugin/Derivative/LocalTasks.php]: Yes, that's a good idea. I switched the logic around to save that exclamation mark and my logic processing brain.

**Comment by seamuslee001** [comment]
looks good to me

**Comment by adixon** [comment]
Thanks!

**Pull request merged**